hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/ReplicationServer.java

```diff
@@ -177,6 +179,11 @@ public static final class ReplicationConfig {
     static final String REPLICATION_OUTOFSERVICE_FACTOR_KEY =
         PREFIX + "." + OUTOFSERVICE_FACTOR_KEY;
 
+    public static final String ZEROCOPY_ENABLE_KEY = "zerocopy.enabled";
+    private static final boolean ZEROCOPY_ENABLE_DEFAULT = false;
```

Comment:
Any reason not to enable this by default? It will be better for testing if we turn it on in this PR.